### PR TITLE
Add ratings typings

### PR DESCRIPTION
**Pull request by Matthew5002**
Fixed a number of dependencies + references + recovered missing file
Added typings for business FutureProof rating breakdown + certificates
Added documentation for ratings folder views.
Made certificates and ratings breakdown modular (input array of certificates/ratings)

### CODE REVIEW

File: packages/client/src/components/ratings/FutureProofRatingView.tsx [hunk 1]
```diff
@@ -2,52 +2,33 @@ import CircularRatingIndicator from '@components/ratings/CircularRatingIndicator
...
-  }
-]
+type ComponentProps = Pick<
+  DisplayableBusiness,
+  'sustainabilityScore' | 'sustainabilityCertificates' | 'sustainabilityRatings'
+>
 
-const FutureProofRatingView = ({ businessToDisplay } : { businessToDisplay : DisplayableBusiness }) => {
+/**
+ * This function creates the whole FutureProof rating view, including certificates and individual ratings breakdown.
```
Comment (bridges-wood) [inline packages/client/src/components/ratings/FutureProofRatingView.tsx:14]: This could be more descriptive

**Review by bridges-wood — CHANGES_REQUESTED** [review]
Excellent work. Couple of tiny things, and if you bring your branch up to date with @choobs1 's PR, then we'll be good to go.

**Review by bridges-wood — CHANGES_REQUESTED** [review]
Excellent work. Couple of tiny things, and if you bring your branch up to date with @choobs1 's PR, then we'll be good to go.

**Review by bridges-wood — CHANGES_REQUESTED** [review]
Excellent work. Couple of tiny things, and if you bring your branch up to date with @choobs1 's PR, then we'll be good to go.

File: packages/client/src/components/BusinessViews/ImagesCarousel.spec.tsx [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+const Image2 =
+   'https://content-prod-live.cert.starbucks.com/binary/v2/asset/137-42658.jpg'
+const Image3 =
+   'https://cdn.vox-cdn.com/thumbor/1WVT8VSapMXbHPvsTEPxZp2gUrk=/0x0:1347x897/1200x800/filters:focal(567x342:781x556)/cdn.vox-cdn.com/uploads/chorus_image/image/62192379/starbucksredcups2015.1541431580.jpg'
+const Image4 =
+   'https://cdn.vox-cdn.com/thumbor/bSoWzne8VZvz8tavhebsL7DNik0=/0x0:5860x4008/1200x800/filters:focal(3243x1967:4179x2903)/cdn.vox-cdn.com/uploads/chorus_image/image/67132574/shutterstock_1410002591.0.jpg'
+
 it('matches snapshot', () => {
-  const { toJSON } = render(<ImagesCarousel />)
+  const { toJSON } = render(<ImagesCarousel Image1={Image1} Image2={Image2} Image3={Image3} Image4={Image4}/>)
```
[the remark this answers is not in the document]
Comment (Matthew5002) [reply packages/client/src/components/BusinessViews/ImagesCarousel.spec.tsx:15]: That was something Cam was working on so I don't want to mess around with it and have even more merge issues. (Agreed though haha)

**Review by bridges-wood — APPROVED** [review]
LGTM

**Review by choobs1 — COMMENTED** [review]
LETS GO NO MORE BABEL PROBLEMS?!! 🙌

**Pull request merged**